client/webserver/api.go

```diff
@@ -1153,6 +1147,48 @@ func (s *WebServer) apiUser(w http.ResponseWriter, r *http.Request) {
 	writeJSON(w, response, s.indent)
 }
 
+// apiGetNotes handles the 'getnotes' API request.
+func (s *WebServer) apiGetNotes(w http.ResponseWriter, r *http.Request) {
+	notes, err := s.core.GetNotifications(100)
+	if err != nil {
+		s.writeAPIError(w, err)
+		return
+	}
+	response := struct {
+		Notes []*db.Notification `json:"notes"`
+		OK    bool               `json:"ok"`
+	}{
+		Notes: notes,
+		OK:    true,
+	}
+	writeJSON(w, response, s.indent)
+}
+
+// apiAuthDexConnections handles the 'authdexconns' API request.
```

Comment:
Maybe state when this would be used.  The use case is... importing an account but not needing to re-login to dexc?

Depending on the answer, that might imply that `apiAccountImport` should be updated to make the `(*Core).AuthDEXConnections` call rather than troubling the frontend with sequential api requests.

Reply:
Or perhaps `(*Core).AccountImport` should be doing `(*Core).initializeDEXConnections` and there's no need at all for `(*Core).AuthDEXConnections`?   It's not clear when a consumer would want to use `AuthDEXConnections` more generally.

Reply:
I've removed `AuthDexConnections`. The only thing that is missing now that we had before, is users of `dexcctl` seeing which servers the client was able to successfully connect to at login. They also saw a huge list of 100 notifications previously, which was a bit distracting.